parts/chapters/subsections/12.3/RPTSCHED.fodt

```diff
@@ -11813,7 +11813,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
            7wkAAAAASUVORK5CYII=
           </office:binary-data>
          </draw:image>
-        </draw:frame><text:span text:style-name="T4073"><text:line-break/></text:span>Figure <text:sequence text:ref-name="refFigure99" text:name="Figure" text:formula="ooow:Figure+1" style:num-format="1">12.14</text:sequence>: RPTSCHED: <text:s/><text:span text:style-name="T6872">WELSPECS</text:span> - Multi Segment Well Connection Sub-Report</text:p>
+        </draw:frame><text:span text:style-name="T4073"><text:line-break/></text:span>Figure <text:sequence text:ref-name="refFigure99" text:name="Figure" text:formula="ooow:Figure+1" style:num-format="1">12.14</text:sequence>: <text:a xlink:href="#__RefHeading___Toc268459_1366622701">RPTSCHED</text:a>: <text:s/><text:span text:style-name="T6872"><text:a xlink:href="#__RefHeading___Toc268463_1366622701">WELSPECS</text:a></text:span> - Multi Segment Well Connection Sub-Report</text:p>
```

Comment:
Note links have been created in the Figure captions. Should we keep these?

Reply:
If we shall remove links in Figure captions with the script, I see one problem: The captions are not inside any special XML element that the script can check for. However, I think it always will be nested inside a `draw:frame`. So we could let the script check for that element instead? Hopefully there will be no false positives where `draw:frame` is used in other ways than creating a Figure, and where we want the script to link text..

Reply:
> it always will be nested inside a draw:frame. So we could let the script check for that element instead?

See #441 for an example of this